tests/test_warm_reboot.py

```diff
@@ -60,6 +60,19 @@ def check_port_oper_status(appl_db, port_name, state):
             break
     assert oper_status == state
 
+def check_port_host_tx_ready_status(state_db, port_name, status):
```

Comment:
@noaOrMlnx have you tested in actual switch host_tx_ready is not deleted during warm-reboot? If its not delted, what is preventing it from deleted from STATE_DB's port table? Any code pointers?

Reply:
The table is deleted. 
The thing is, that after warm reboot, host_tx_ready value will be false, and we won't get any new notification from lower layers saying that host_tx_ready=true. 
That's the reason we need to take the current value of host_tx_ready using the new API